### DISCUSSION TITLE
Running openpolicyagent/opa docker on linux/arm64/v8 platform

### DISCUSSION THREAD

**Discussion by nissimsan** [open-policy-agent/community#551] [OPA and Rego] [answered]
Hi there! 

I'm looking to run the OPA HTTP server (https://hub.docker.com/r/openpolicyagent/opa) on a raspberry pi, and hit this error: 

```
sudo docker run -it --rm -p 8181:8181 openpolicyagent/opa run --server --addr :8181
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
exec /opa: exec format error
```

I'm really no docker wiz, so hoping for a hint how I can get the image up and running? 

Noting that this works perfectly fine locally (Mac M1).

Cheers and thanks in advance!

**Comment by anderseknert** [comment, marked as answer]
Hi there 👋 Try using a `-static` version, e.g.

```
sudo docker run -it --rm -p 8181:8181 openpolicyagent/opa:0.61.0-static run --server --addr :8181
```

**Comment by nissimsan** [comment]
Yes! Thanks a ton - perfect. 

... Tack så mycket  :) 🇸🇪🇩🇰

**Reply by anderseknert** [reply]
Det var så lidt!
